src/Tables.jl

```diff
@@ -308,7 +261,7 @@ function columns end
 """
     Tables.rows(x) => Row iterator
 
-Accesses data of input table source `x` row-by-row by returning a [`Row`](@ref) iterator.
+Accesses data of input table source `x` row-by-row by returning a [`Row`](@ref Row) iterator.
```

Comment:
One disadvantage of this PR is that you can't use ``` [`Row`](@ref) ```. Maybe this is fixable on Documenter.jl side, though.